.github/workflows/python-test-suite.yml

```diff
@@ -58,7 +58,15 @@ jobs:
 
       - name: Run the python tests suite
         working-directory: python
-        run: uv run --python ${{ matrix.python-version }} pytest tests -m "not slow"
+        run: |
+          uv run --python ${{ matrix.python-version }} pytest tests -m "not slow" --cov=src/magika --cov-report=xml --cov-report=term
+
+      - name: Upload coverage report to Coveralls
+        if: always()
```

Comment:
this `if` should be removed?

Reply:
The" if: always()"  will ensure that the coverage report is uploaded even if one test case fails. If this is not necessary for our workflow, I can remove it